charts/notification-orchestration/values.yaml

```diff
@@ -0,0 +1,47 @@
+args: []
+command: []
+horizontalPodAutoscaler:
+  enabled: false
+  averageCpuUtilization: 75
+  averageMemoryUtilization: 75
+image:
+  name: "ghga/notification-service"
+podAnnotations: {}
+resources:
+  limits:
+    cpu: "500m"
+    memory: "256M"
+  requests:
+    cpu: "100m"
+    memory: "64M"
+
+config_prefix: ns
```

Comment:
```suggestion
config_prefix: nos
```

Reply:
- [ ] Add also to config in helmfile, it is missing there

Reply:
Fixed it already.